lib/aescrypt.rb

```diff
@@ -55,7 +63,9 @@ def self.key_digest(password)
   #:arg: iv => String
   #:arg: cipher_type => String
   def self.decrypt_data(encrypted_data, key, iv, cipher_type)
-    aes = OpenSSL::Cipher::Cipher.new(cipher_type)
+    self.warn_if_ecb cipher_type
+
+    aes = OpenSSL::Cipher.new(cipher_type)
```

Comment:
Technically, `aes` is not necessarily the right variable name, as `cipher_type` might specify non-AES.